woocommerce/woocommerce-gutenberg-products-block
# Add storybook entry for Label component

## What
Add a storybook entry for the Label component

## Why
To illustrate the component's use and provide some level of documentation.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Open storybook. ensure the External Components -> Label renders
2. Add the All Products and All Reviews blocks to a page
3. For the All Reviews block, make sure the "Load more reviews" setting is enabled and set the starting number of reviews to 1
4. Make sure you have at least 2 reviews on your site. You can go to any product and click the Review tab
5. Make sure the "Load more" button is visible on the front end.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:
